src/hybrid/parse_hybrid.c

```diff
@@ -253,6 +256,73 @@ static int parseRangeClause(ArgsCursor *ac, VectorQuery *vq, ParsedVectorData *p
   }
   return REDISMODULE_OK;
 }
+
+static int parseLinearClause(ArgsCursor *ac, HybridLinearContext *linearCtx, QueryError *status) {
+  // LINEAR 4 ALPHA 0.1 BETA 0.9 ...
+  //        ^
+  long long params;
+  CheckEnd("parameter count");
```

Comment:
Should be consistent with our documentation and use "arguments" instead of "parameters"
As above, give the expected number of missing arguments.